services/app/apps/codebattle/assets/js/widgets/components/Rooms.jsx

```diff
@@ -20,7 +20,7 @@ export default function Rooms({ disabled }) {
           <span className="mr-2">{activeRoom.name}</span>
         </Dropdown.Toggle>
 
-        <Dropdown.Menu>
+        <Dropdown.Menu className="scrollable-menu-dropdown-chat">
```

Comment:
К нашим кастомным классам делают приписку `cb` (СodeBattle)

```suggestion
        <Dropdown.Menu className="h-auto cb-overflow-x-hidden cb-scrollable-menu-dropdown-chat">
```